lstchain/reco/r0_to_dl1.py

```diff
@@ -439,6 +445,19 @@ def r0_to_dl1(
                     add_global_metadata(container, metadata)
 
                 event.r0.prefix = ''
+                if is_simu:
+                    # None values in telescope/image table
+                    writer.exclude(f'telescope/image/{tel_name}', 'image_mask')
+                    writer.exclude(f'telescope/image/{tel_name}', 'parameters')
+                    # None values in telescope/parameters table
+                    writer.exclude(f'telescope/parameters/{tel_name}', 'disp_norm')
+                    writer.exclude(f'telescope/parameters/{tel_name}', 'disp_dx')
```

Comment:
I'm not getting this. Why excluding `disp*` params for simulated data?

Reply:
> I'm not getting this. Why excluding `disp*` params for simulated data?

I think they are not yet calculated at this point in the chain.

Reply:
Ah yes

Reply:
Still not getting why excluding the parameters

Reply:
Probably not needed, in the sense that there would be no warning, right?   On the other hand, having there a parameter which is not yet calculated may be misleading.

Reply:
So excluding them here is correct for observed data. THese are the truth values which can only be calculated on simulated data.

Reply:
Ah wait. The if is wrong, right?

These columns should be excluded `if not is_simu`

Reply:
nope, these columns are indeed not yet reconstructed yet, the addition of disp parameters is at the very end and is done for the whole dataframe, the (in)famous `for name in ['LST_LSTCam']` loop which was fixed recently

Reply:
But then they should be ignored for both `is_simu` and `not is_simu`, right? So the if is not needed?

Reply:
If these are true values they should not be called "disp*". All we need is true gamma direction on camera, and, if you want to store it precalculated, also "dist". The name disp as far as I know is for reconstructed quantities.  No need to change it before release, though!

Reply:
I actually realized, that I didn't test on data, so the data-related warnings were not suppressed. The corresponding fix is added